### PR TITLE
Parse LOCPOT

### PR DESCRIPTION
**Pull request by shyamd**
Simple addition to the VASP drone to parse LOCPOT and save the axis averages which can be used for surface and defect corrections. 

Currently defaults to not parse, but can very easily change to parse by default since it's a tiny amount of data that might be useful for other things?

### CODE REVIEW

File: atomate/vasp/tests/test_drones.py [hunk 1]
```diff
@@ -126,5 +128,19 @@ def test_detect_output_file_paths(self):
                               'wavecar': 'WAVECAR.relax1.gz'},
                              doc['calcs_reversed'][1]['output_file_paths'])
 
+    def test_parse_locpot(self):
+        drone = VaspDrone(parse_locpot=True)
+        doc = drone.assimilate(self.Si_static)
+
+        self.assertTrue(drone.parse_locpot)
+        self.assertTrue('locpot' in doc['calcs_reversed'][0]['output'])
+        self.assertTrue(0 in doc['calcs_reversed'][0]['output']['locpot'])
```
Comment (computron) [inline atomate/vasp/tests/test_drones.py]: are these 3 checks needed? seems like the tests afterwards which reference these indices will automatically check for this?

Comment (computron) [reply atomate/vasp/tests/test_drones.py]: (referring to checks of form ``self.assertTrue(0 in doc['calcs_reversed'][0]['output']['locpot'])``

**Comment by computron** [comment]
For parse by default, can you give me an idea of:

* how much data is it? (is it just the 3 numbers or more than that)
* does it / can it take a long time to parse?

File: atomate/vasp/tests/test_drones.py (hunk 1 above)
Comment (shyamd) [reply atomate/vasp/tests/test_drones.py]: You're right that the next three tests cover any issues with the indicies.  These three don't hurt and provide more explicit checks for what is wrong.

**Comment by shyamd** [comment]
The LOCPOT grid is based on NGF, and from a cursory review of calcs i've run that ranges from 30-40 to as much as 100 numbers per axial direction, as much as 3-4 kb of floats added to the document.

**Comment by computron** [comment]
Ok I'm just going to merge as-is for now. If you feel strongly that the default should be True, you should submit another PR and I'll likely just merge w/o further discussion.

**Pull request merged**